CUSTOM_BUNDLE.md

```diff
@@ -1,13 +1,31 @@
 # Custom bundle
 You can simply make custom bundles yourself, if none of the [distributed packages](https://github.com/plotly/plotly.js/blob/master/dist/README.md) meet your needs, or you want to make a more optimized bundle file with/without specific traces and transforms.
 
-Install plotly.js, move to plotly.js folder then install plotly.js dependencies:
+Clone plotly.js, where the <version> is one of [these](https://github.com/plotly/plotly.js/tags):
+```sh
+git clone --depth 1 --branch <version> https://github.com/plotly/plotly.js.git
+```
+
+Move to plotly.js folder then install plotly.js dependencies:
+```sh
+cd plotly.js
+```
+
+Make sure you have the versions of node/npm that's recommended in the [./CONTRIBUTING.md](./CONTRIBUTING.md#prerequisites) list of Prerequisites.
+
+To download a specific node version look [here](https://nodejs.org/en/download/package-manager).
+
+Install the dependencies
 ```sh
-npm i plotly.js
-cd node_modules/plotly.js
 npm i
 ```
 
+Note: You can always switch to another version with:
```

Comment:
Is it still working if they used `--depth 1` and in case they want to go behind the version they initially cloned?

Reply:
I've added a not on --depth 1 for CI